app/src/main/java/com/assist_software/expenseappmvp/data/database/dao/UserDao.kt

```diff
@@ -20,4 +20,7 @@ interface UserDao {
 
     @Query("DELETE FROM users WHERE userId = :id")
     fun deleteUser(id: Long = 0)
+
+    @Query("SELECT userId FROM users WHERE userEmail = :email LIMIT 1")
```

Comment:
I think you don't need to set LIMIT if you have userEmail field set as index unique

Reply:
Solved